demos/browser/app/meeting/meeting.ts

```diff
@@ -711,12 +712,7 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
           if (signalStrength !== null) {
             this.roster[attendeeId].signalStrength = Math.round(signalStrength * 100);
           }
-          if (!this.roster[attendeeId].name) {
-            const response = await fetch(`${DemoMeetingApp.BASE_URL}attendee?title=${encodeURIComponent(this.meeting)}&attendee=${encodeURIComponent(attendeeId)}`);
-            const json = await response.json();
-            const name = json.AttendeeInfo.Name;
-            this.roster[attendeeId].name = name ? name : '';
-          }
+          this.roster[attendeeId].name = externalUserId.split('#')[1];
```

Comment:
Do we need to do the same changes like in meetingV2?

Reply:
There is a similar change in meetingV2:

```
      if (!this.roster[attendeeId]) {
        this.roster[attendeeId] = {
          name: externalUserId.split('#')[1] + (isContentAttendee ? ' «Content»' : ''),
        };
      }
```

I did the most extensive cleanup to `meetingV2` and only fixed `meeting` for backwards compatibility hence the smaller and slightly different change set in `meeting`.